src/api/views/domain_views.py

```diff
@@ -189,10 +189,10 @@ def post(self, domain_id):
         domain = domain_manager.get(document_id=domain_id)
 
         domain_name = domain["name"]
-        category = request.args.get("category")
+        template_name = request.args.get("template-name")
```

Comment:
Can this arg be passed as "template_name" instead of "template-name", that gets really confusing if we're required underscores in some places and dashes in others.